homeassistant/config_entries.py

```diff
@@ -1465,14 +1464,12 @@ def _async_abort_entries_match(
     if match_dict is None:
         match_dict = {}  # Match any entry
     for entry in other_entries:
-        if all(
-            item
-            in ChainMap(
-                entry.options,  # type: ignore[arg-type]
-                entry.data,  # type: ignore[arg-type]
-            ).items()
-            for item in match_dict.items()
-        ):
+        options_items = entry.options.items()
+        data_items = entry.data.items()
+        for kv in match_dict.items():
+            if kv not in options_items and kv not in data_items:
```

Comment:
Won't this mean that we'll raise if one item is matched in options and the other item is matched in data for a two item match? I think we want to only raise if either all items are matched in options or all items are matched in data.

Reply:
I guess it's the same behavior as before though, so we shouldn't change it.

Reply:
That was my first thought as well when I saw this. I didn't change it since I thought it was likely to be a breaking change 👍